sdk/eventgrid/Microsoft.Azure.EventGrid/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 3.3.0-preview.3 (Unreleased)
-
+## 3.2.1 (2021-06-15)
+Update package description to include pointer to the new Azure.Messaging.EventGrid package and the migration guide.
```

Comment:
hehe... first changelog entry is to say that we are adding a pointer to something else